src/components/molecules/EditorTitleBar/EditorTitleBar.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import {Button} from 'antd';
+
+import {useAppDispatch} from '@redux/hooks';
+import {toggleLeftMenu} from '@redux/reducers/ui';
+
+import {EditorMonoPaneTitle} from '@atoms';
+
+import * as S from './EditorTitleBar.styled';
+
+interface IProps {
+  title: string;
+  leftButtons?: React.ReactNode;
+  closable?: boolean;
+  children?: React.ReactNode;
+}
+
+const EditorTitleBar: React.FC<IProps> = props => {
```

Comment:
We should use the `TitleBar` from inside `@monokle-core/components` instead of creating our own

Reply:
Why would the `TitleBar` need to be updated? It should already be implemented to support all needs, or is there something missing?

Reply:
It looks brand new I dont suppose it can support these changes mate because I added some new classes on tab that is not used before and without those can not update new tab styles I want to share the new look but I could not find the ticket 2756

Reply:
Found it :) https://github.com/kubeshop/monokle/issues/2756